src/components/PageTemplate/Footer.jsx

```diff
@@ -1,23 +1,34 @@
 import React from 'react';
 import {Flex, Link, Text} from "@chakra-ui/react";
 import {Icon} from "@chakra-ui/icons";
-import {FaEnvelope, FaGithub} from 'react-icons/fa';
+import {FaGithub, FaFacebookSquare, FaEnvelope} from 'react-icons/fa';
+
+const MARGIN = 2
 
 const Footer = () => (
     <Flex direction="column" p={16} align="center">
+        <Flex align="center">
+            <Link href="/about" m={MARGIN}>About Us</Link>
+            <Link m={MARGIN}>Help</Link>
+        </Flex>
         <Flex align="center">
             <Text m={1}>Connect with us:</Text>
             <Link href="https://github.com/InvestoBull/stock-price-prediction"
-                  isExternal={true}
+                  isExternal
+                  p={0} bg="transparent" _hover={{color: "brand.500"}}>
+                <Icon as={FaGithub} w={6} h={6} m={MARGIN}/>
+            </Link>
+            <Link href="https://www.facebook.com/people/Investobull-Investors/100068350916511/"
+                  isExternal
                   p={0} bg="transparent" _hover={{color: "brand.500"}}>
-                <Icon as={FaGithub} w={6} h={6} m={2}/>
+                <Icon as={FaFacebookSquare} w={6} h={6} m={MARGIN}/>
```

Comment:
Code repitition 
Would be a nice idea to declare a custom Icon on the top as a component and specify the common fields - w, h, m there and send as an as other prop

Reply:
will fix 👌🏼